ui-spacetimechart/src/components/workSchedules/WorkSchedules.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import Region from './Region';
+import { type WorkSchedule } from '../../types';
+
+type WorkSchedulesProps = {
+  workSchedules: WorkSchedule[];
+  pattern: string;
+};
+/**
+ * Display the workschedule projection on the Space time chart.
+ * a workschedule can contains several track occupied during a period of this
```

Comment:
- several typos
- I think your sentence is not finished ? (this -> time ?)
```suggestion
 * A workschedule can contain several tracks occupied during a period of this
```

Reply:
done